docs/permissions/plugin-authors/03-adding-a-resource-permission-check.md

```diff
@@ -80,9 +82,7 @@ router.put('/todos', async (req, res) => {
   const decision = (
     await permissions.authorize(
       [{ permission: todoListUpdatePermission, resourceRef: req.body.id }],
-      {
-        token,
-      },
+      credentials,
```

Comment:
```suggestion
      { credentials },
```

Still an options object 😅

Reply:
That is interesting, I don't get any type error, even before when I was passing the token...

Reply:
But if I apply the suggestion I get this error:
![image](https://github.com/user-attachments/assets/d9dc2234-fea1-4f02-b111-6ac0e01b3eb0)

Reply:
hmm, are you perhaps testing with the client-side permissions API? Is the `permissions` client you got the implementing `PermissionsService`?

Reply:
I'm using `coreServices.permissions`, is that right 🤔 ?

Reply:
I was using a wrong type in the router options, it should be fixed now 🙏🏻 .